src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,11 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+    const digits = String(number).split('');
+    const reversedDigits = digits.reverse();
+    return reversedDigits.map(digit => Number(digit));
+    
+  
+  const reversed = numberToReversedDigits(12345);
+  console.log(reversed);
+};
```

Comment:
So it looks like youre doing something quite interesting here known as recursion, where a function calls itself! It turns out you arent as you are returning a value on line 4 which prevents the function call on line 7 from ever being reached. The logic here is good but you can boil it down a little bit further by chaining the methods together:
```
  String(number)
    .split("")
    .reverse()
    .map((num) => Number(num));
```